package.json

```diff
@@ -33,13 +33,13 @@
   },
   "dependencies": {
     "graceful-fs": "~4.1.2",
-    "minimatch": "~0.2.12",
-    "readable-stream": "~1.0.26-2"
+    "minimatch": "^2.0.10",
```

Comment:
Pleas no carets, I don't trust minor versions _yet_.
So updating the versions is fine but keep the `~` range specifier. for all of these.
To have `npm install --save` do that for you do `npm config set save-prefix='~'`.

This applies to all dependencies including dev-dependencies.